tests/codegen/issues/issue-126585.rs

```diff
@@ -0,0 +1,23 @@
+//@ compile-flags: -Copt-level=s
+//@ min-llvm-version: 19
+
+// Test for #126585.
+// Ensure that this IR doesn't has extra undef phi input, which also guarantees that this asm
+// doesn't has subsequent labels and unnecessary `jmp` instructions.
```

Comment:
Drive-by spellchecking 😄 

```suggestion
// Ensure that this IR doesn't have extra undef phi input, which also guarantees that this asm
// doesn't have subsequent labels and unnecessary `jmp` instructions.
```

Reply:
Oops, thanks!